### ISSUE TITLE
💡[Feature]: Types of activation function in Artificial neural network (ANN) from scratch and visualised

### ISSUE THREAD

**Issue by Shantnu-singh** [recodehive/machine-learning-repos#974]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

This feature will cover the mathematical formulations, properties, and practical applications of activation functions such as sigmoid, tanh, ReLU, Leaky ReLU, and softmax, along with visualizations and code snippets for implementation.

### Use Case

This feature is beneficial for students, educators, and practitioners in the field of deep learning. Students can use the comprehensive documentation to understand the role and impact of different activation functions on neural network performance.

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue
